sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollectionViews.java

```diff
@@ -233,7 +233,7 @@ public T apply(Iterable<WindowedValue<T>> contents) {
 
     @Override
     public Iterable<T> apply(Iterable<WindowedValue<T>> contents) {
-      return Iterables.unmodifiableIterable(
+      return ImmutableList.copyOf(
```

Comment:
<!--new_thread; commit:a2ad4c5858521a9f42105f87fa3401fbab5bff18; resolved:0-->
Can't do this - the contents might be an entire PCollection streamed in.